source/conditionals.rst

```diff
@@ -61,14 +68,20 @@ value to be equal:
     >>> 5 == '5'
     False
 
+Questions
+-------
+1. Write some equality statements of your own
+
+2. Instead of == you can also use `!=` `<=` `<` `>=` or `>`. Write some equality statements using these
```

Comment:
I wonder if we should have an 'extra' part about strict comparison in JS - even just a link in the appendix or further learnings somewhere?